fix: d266 chart updated

vak2ve opened the pull request:
@joewiz table restructured to better match page image

joewiz commented:
Could you post a screenshot, to facilitate comparison to those in https://github.com/HistoryAtState/frus/issues/333?

vak2ve commented:
Actions taken:

1. Removed extraneous 5th column 
2. Removed extra column headings through (intended to correspond with repeated headings after page breaks)
3. Removed incorrect row spans that affected formatting
4. Added new rendition values to ensure first column (time) was readable at the same level as the text of the other columns and that no breaks occurred in that first column

<img width="626" alt="Screenshot 2024-11-22 at 11 50 47 AM" src="https://github.com/user-attachments/assets/57097c52-0137-4291-9997-9f239416bbb5">

<img width="606" alt="Screenshot 2024-11-22 at 11 51 18 AM" src="https://github.com/user-attachments/assets/95ab519e-19f6-4426-86aa-6562ac79136b">
